spec/fixtures/basic_question.yml

```diff
@@ -1,4 +1,15 @@
 ---
+location:
```

Comment:
Is it worth splitting this out into a new fixture? It now contains *three* questions, so is no longer just a 'basic question'.

Reply:
I could remove the final question, and then this is the smallest valid question?

Reply:
That would be better :+1:

Reply:
Ideally, it should be possible to overwrite the mandatory questions array in tests, so that the basic question is still valid. However, I'm not going to worry about this for now.